### ISSUE TITLE
Classification task

### ISSUE THREAD

**Issue by iaixuexi** [AlexeyAB/darknet#4523]
I want to do classification tasks on the darknet53.conv.74 pre-trained model, but when I set learning_rate = 0.001, the loss value is almost 0 at 10000 generations. What is wrong with this?

**Comment by AlexeyAB** [comment]
You are doing something wrong

**Comment by iaixuexi** [comment]
But I only changed the output of the last layer in the darknet53.cfg file. I followed this command: ./darknet classifier train cfg/obj.data cfg/darknet53.cfg backup/darknet53.conv.74. I don't know what's wrong with this, do you know what might be wrong here? thank you
